bosesoundtouch.js

```diff
@@ -364,14 +587,60 @@ class boseSoundTouch {
                 }
                 this.setState(BOSE_ID_ZONES_MASTER_OF, members.join(';'), {ack: true});
                 this.setState(BOSE_ID_ZONES_MEMBER_OF, '', {ack: true});
+				//Find out, if all speakers are members (playEverywhere)
+				this.adapter.getObjectView(
+					'system', 'instance', {
+						startkey: 'system.adapter.bosesoundtouch.',
+						endkey: 'system.adapter.bosesoundtouch.\u9999' },
+					function (err, doc) {
+						if (doc && doc.rows && doc.rows.length) {
+							var toDoList = [];
+							for (var i = 0; i < doc.rows.length; i++) {
+								var obj = doc.rows[i].value;
+								if (obj.native.address != instance.adapter.ipAddress) {
+									var systemId  = doc.rows[i].id.split('.');
+									var index = systemId[systemId.length - 1];
+									var namespace = systemId[systemId.length - 2] + '.' + index + '.';
+									toDoList.push(namespace + BOSE_ID_INFO_IP_ADDRESS.id);
+									toDoList.push(namespace + BOSE_ID_INFO_MAC_ADDRESS.id);
+								}
+							}
+							instance._collectPlayEverywhereData(toDoList, instance._collectPlayEverywhereDataFinished_checkPlayEverywhere, members);
+						}
+						else {
+							instance.adapter.log.debug('No objects found: ' + err);
+						}
+					}
+				);
             }
             else {
                 this.setState(BOSE_ID_ZONES_MASTER_OF, '', {ack: true});
                 this.setState(BOSE_ID_ZONES_MEMBER_OF, obj.master, {ack: true});
```

Comment:
you missed the $ here like on top 
obj.$.master, I found the same issue. Not sure why this change is needed. but I see it on my bose different too.